src/components/carData/carDataSecondary.java

```diff
@@ -0,0 +1,30 @@
+package components.carData;
+
+/**
+ * Contains the secondary methods for the carData Component.
+ */
+public abstract class CarDataSecondary implements CarData {
+
+    /**
```

Comment:
You need to include at least some of the common methods here (i.e., toString, hashCode, and/or equals).